src/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from 'react'
+import { useState } from 'react'
+import {
+  DropdownContainer,
+  DropdownHeader,
+  DropdownContent,
+  DropdownItemContainer,
+  DropdownHeaderContainer,
+} from './Dropdown.style'
+import { DropdownProps } from './interface'
+
+const Dropdown: React.FC<DropdownProps> = ({ children }) => {
+  const options = children
```

Comment:
misalkan mau kasih alias ke variable, lebih baik begini

```suggestion
const Dropdown: React.FC<DropdownProps> = ({ children: options }) => {
```